client/src/locales/en/translation.json

```diff
@@ -59,16 +59,22 @@
             "CompareBTN":"Compare",
             "All":"All",
             "labels": {
-                "Country":"Country",
-                "Employee":"Employee"
+                "Employee":"Employee",
+                "ClientType": "Client Type",
+                "Country":"Country"
```

Comment:
Move the country key above ClientType, it's just to make the disposition between the two files match.